servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ServerContext.java

```diff
@@ -50,6 +50,35 @@ default void awaitShutdown() {
         awaitTermination(onClose().toFuture());
     }
 
+    /**
+     * Toggles the server's channel accepting connection ability.
+     * <p>
+     * Passing a {@code false} value, will stop accepting connections on the server channel, without affecting any other
+     * interaction to currently open child channels (i.e., reads / writes).
+     * <p>
+     * Depending on the transport, connections may still get ESTABLISHED, see.
+     * {@link ServiceTalkSocketOptions#SO_BACKLOG backlog} and
+     * <a href="https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt">SOMAXCONN</a>.
```

Comment:
Perhaps worth highlighting it's a Linux specific example, for instance by
```
    * (...) and (in case of Linux)
    * <a href="https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt">SOMAXCONN</a>.
```